_package/xmsinterp/interpolate/interp_linear.py

```diff
@@ -0,0 +1,383 @@
+"""
+********************************************************************************
+* Name: interp_linear.py
+* Author: Gage Larsen, Matt LeBaron
+* Created On: April 29th, 2019
+* Copyright: (c)
+* License: BSD 2-Clause
+********************************************************************************
+"""
+
+from .._xmsinterp.interpolate import InterpLinear as iLin
+
+
+class InterpLinear:
```

Comment:
Inherit from object for Python 2 support.

Reply:
fixed in commit 501ce2a